tests/test_req2flatpak.py

```diff
@@ -7,8 +7,14 @@
 from itertools import product
 from pathlib import Path
 from typing import List
+from unittest import skipUnless
 from unittest.mock import patch
 
+try:
+    import yaml
+except ImportError:
+    yaml = None  # type: ignore
```

Comment:
Not sure whether there is a more elegant solution.

Reply:
looks ok to me, easy to understand